producer.go

```diff
@@ -99,7 +98,13 @@ func (p *Producer) consumeDeliveries() {
 			}
 		} else {
 			p.log.Printf("Error consuming delivery event: Unknown event type (%s)", ev)
-			atomic.AddUint64(&stats.producerErr, 1)
+
+			// Ignore 'connection reset by peer' errors since they manifest because
+			// of the connection reaper.
+			// See https://github.com/edenhill/librdkafka/wiki/FAQ#why-am-i-seeing-receive-failed-disconnected.
+			if !strings.Contains(ev.String(), "Connection reset by peer") {
```

Comment:
For readability and a more idiomatic version of this, consider the following:

```go
// ...
} else {
  if string.Contains(ev.String(), "Connection reset by peer") {
    continue
  }

  // log
  // increment counter
}
```

Reply:
OK for the continue idiom, but I still want to log this.